pandas/tests/dtypes/test_inference.py

```diff
@@ -415,6 +420,29 @@ def test_isneginf_scalar(self, value, expected):
         result = libmissing.isneginf_scalar(value)
         assert result is expected
 
+    @pytest.mark.parametrize(
+        "convert_to_nullable_boolean, exp",
+        [
+            (
+                True,
+                BooleanArray(
+                    np.array([True, False], dtype="bool"), np.array([False, True])
+                ),
+            ),
```

Comment:
do we have tests for maybe_convert_bool already? can you integrate with those

Reply:
No. maybe_convert_bool is only used by python parser.